src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir , serif;
+  font-weight: 400;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+/* position for big box & icon */
```

Comment:
better not leave comments in your code. From the names of your selectors should be easy to get what they are actually styling